### ISSUE TITLE
Draft — Updated Invalid date 

### ISSUE THREAD

**Issue by kodeartist** [writingink/wink#244]
When I want to save a post I am experiencing this issues. Is this a problem? I have done 2 re-installs with the same issue.
@themsaid Any reason?

**Comment by MagedAhmad** [comment]
i was here to report the same issue
the response is 422 with 
```
{"message":"The given data was invalid.","errors":{"publish_date":["The publish date is not a valid date."]}}
```

**Comment by kodeartist** [comment]
@MagedAhmad I tried with laravel 6 and it seems it works. Maybe it is a Laravel 7.* Issue!

**Comment by oliverbj** [comment]
I also encounter this issue, also running Laravel 7+ (PHP 7.4)

**Comment by kodeartist** [comment]
@oliverbj did you get issue solved?

**Comment by cedaesca** [comment]
I'm experiencing this issue on Laravel 7.9.2

**Comment by ibrunotome** [comment]
Hello @themsaid, is there the possibility to open the package to new maintainers? Maybe someone can help with the pull requests and other bugs like this.

**Comment by MagedAhmad** [comment]
@kodeartist works fine on laravel 6.x, give it a try .

**Comment by surazz501** [comment]
I am also having same issue on laravel 7.10.1

**Comment by themsaid** [comment]
Fixed in 1.0. releasing soon. Thanks everyone :)
